### DISCUSSION TITLE
doc/info/%E5%8A%9F%E8%83%BD%E4%BB%8B%E7%BB%8D

### DISCUSSION THREAD

**Discussion by giscus** [zuihou/zuihou-admin-cloud#353] [企业版]
# doc/info/%E5%8A%9F%E8%83%BD%E4%BB%8B%E7%BB%8D

微服务中后台快速开发平台

https://tangyh.top/doc/info/%E5%8A%9F%E8%83%BD%E4%BB%8B%E7%BB%8D.html

**Comment by Tamrac-web** [comment]
文档旧了，需要更新
